src/FormMigration/Steps/FormFieldManager.php

```diff
@@ -28,7 +29,12 @@ public function process()
         if (!$formFields) {
             return;
         }
-        
+
+        $this->fieldBlockRelationships['give-amount'] = [
+            'field' => [], // This property intentionally left empty.
```

Comment:
Can you explain why this is intentional? i'm just confused what's going on here.

Reply:
Fair point. I'll either update the comment or might be able to just remove it.

Reply:
Hopefully this clarifies, see https://github.com/impress-org/givewp/pull/7030/commits/0486e18ffdfbe654c0506120e1470927bba445e7.